### ISSUE TITLE
3.7.2 tests fail on Cygwin

### ISSUE THREAD

**Issue by matzeri** [qhull/qhull#55]
It does not happen on 3.7.1

Two tests are failing as Windows platform the file system is usualy case insensitive
and the binary qhull (aka qhull.exe) and the directory Qhull are seen from cygwin as the same

```
3/11 Testing: smoketest
3/11 Test: smoketest
Command: "/usr/bin/sh.exe" "-c" "./rbox D4 | ./qhull Tv"
Directory: /cygdrive/d/cyg_pub/devel/qhull/test2/qhull-2019.1-1.x86_64/build
"smoketest" start time: Dec 17 08:30 CET
Output:
----------------------------------------------------------
/usr/bin/sh: ./qhull: Is a directory

Starting the rbox smoketest for qhull.  An immediate failure indicates
that non-reentrant rbox was linked to reentrant routines.  An immediate
failure of qhull may indicate that qhull was linked to the wrong
qhull library.  Also try 'rbox D4 | qhull T1'
<end of output>
Test time =   1.07 sec
----------------------------------------------------------
Test Failed.
"smoketest" end time: Dec 17 08:30 CET
"smoketest" time elapsed: 00:00:01
----------------------------------------------------------
4/11 Testing: rbox-10-qhull
4/11 Test: rbox-10-qhull
Command: "/usr/bin/sh.exe" "-c" "./rbox 10 | ./qhull Tv"
Directory: /cygdrive/d/cyg_pub/devel/qhull/test2/qhull-2019.1-1.x86_64/build
"rbox-10-qhull" start time: Dec 17 08:30 CET
Output:
----------------------------------------------------------
/usr/bin/sh: ./qhull: Is a directory
<end of output>
Test time =   0.13 sec
----------------------------------------------------------
Test Failed.
"rbox-10-qhull" end time: Dec 17 08:30 CET
"rbox-10-qhull" time elapsed: 00:00:00
----------------------------------------------------------
```

the problem is located at 

 ```
write_basic_package_version_file(
      "${CMAKE_CURRENT_BINARY_DIR}/Qhull/QhullConfigVersion.cmake"
```
and following lines.
A possible solution is to use a different directory name, as

```
write_basic_package_version_file(
      "${CMAKE_CURRENT_BINARY_DIR}/Config/QhullConfigVersion.cmake"
```

**Comment by matzeri** [comment]
With attached patch the build and tests pass as expected

[qhull-2019.1-1.src.patch.gz](https://github.com/qhull/qhull/files/3972891/qhull-2019.1-1.src.patch.gz)

**Comment by cbbarber** [comment]
Previously fixed on github.  To be released as Qhull 2020.1

write_basic_package_version_file(
    "${CMAKE_CURRENT_BINARY_DIR}/QhullExport/QhullConfigVersion.cmake"
    VERSION ${qhull_VERSION}
    COMPATIBILITY AnyNewerVersion
)

Thanks for reporting the issue and suggesting a fix.

      --Brad
